plugins/API/API.php

```diff
@@ -417,7 +417,8 @@ public function get($idSite, $period, $date, $segment = false, $columns = false)
             if ($mergedDataTable === false) {
                 $mergedDataTable = $dataTable;
             } else {
-                $this->mergeDataTables($mergedDataTable, $dataTable);
+                $merger = new MergeDataTables();
+                $merger->mergeDataTables($mergedDataTable, $dataTable);
```

Comment:
Needed same logic in Goals\API.php but did not want to move it to core for now. It's kinda specific to numeric metrics etc.